azure/03-cluster-configuration/config.ts

```diff
@@ -1,38 +1,22 @@
 import * as pulumi from "@pulumi/pulumi";
 
-let pulumiConfig = new pulumi.Config();
+const pulumiConfig = new pulumi.Config();
 
 // Existing Pulumi stack reference in the format:
 // <organization>/<project>/<stack> e.g. "myUser/myProject/dev"
 
-/*
 const identityStackRef = new pulumi.StackReference(pulumiConfig.require("identityStackRef"));
 const infraStackRef = new pulumi.StackReference(pulumiConfig.require("infraStackRef"));
-*/
 
 export const config = {
-    /*
+    // Resource Group
+    resourceGroupName:        identityStackRef.getOutput("resourceGroupName"),
+
     // Identity
-    adminsIamRoleArn:        identityStackRef.getOutput("adminsIamRoleArn"),
-    devsIamRoleArn:          identityStackRef.getOutput("devsIamRoleArn"),
-    stdNodegroupIamRoleArn:  identityStackRef.getOutput("stdNodegroupIamRoleArn"),
-    perfNodegroupIamRoleArn: identityStackRef.getOutput("perfNodegroupIamRoleArn"),
+    adApplicationId:          identityStackRef.getOutput("adApplicationId"),
+    adSpPassword:             identityStackRef.getOutput("adSpPassword"),
 
     // Infrastructure / Networking
-    vpcId: infraStackRef.getOutput("vpcId"),
-    publicSubnetIds: infraStackRef.getOutput("publicSubnetIds"),
-    privateSubnetIds: infraStackRef.getOutput("privateSubnetIds"),
-    */
-
-    /*
-    defaultVpcId: infraStackRef.getOutput("defaultVpcId"),
-    defaultPublicSubnetIds: infraStackRef.getOutput("defaultPublicSubnetIds"),
-    defaultPrivateSubnetIds: infraStackRef.getOutput("defaultPrivateSubnetIds"),
-     */
-
-    /*
-    existingVpcId: infraStackRef.getOutput("existingVpcId"),
-    existingPublicSubnetIds: infraStackRef.getOutput("existingPublicSubnetIds"),
-    existingPrivateSubnetIds: infraStackRef.getOutput("existingPrivateSubnetIds"),
-     */
+    subnetId:                 infraStackRef.getOutput("subnetId"),
```

Comment:
Can we add code examples in this config to support the default VPC and an existing VPC as pointed out in the comment above for the networking stack to add those in?